src/sentry/static/sentry/app/views/releasesV2/detail/filesChanged/index.tsx

```diff
@@ -65,14 +82,35 @@ class FilesChanged extends AsyncView<Props, State> {
     return (
       <ContentBox>
         {fileList.length ? (
-          Object.keys(filesByRepository).map(repository => (
-            <RepositoryFileSummary
-              key={repository}
-              repository={repository}
-              fileChangeSummary={filesByRepository[repository]}
-              collapsable={false}
-            />
-          ))
+          <React.Fragment>
+            {Object.keys(filesByRepository).map(repository => (
```

Comment:
nb: you could use `Object.entries()` since you use key + value

Reply:
Good idea 👍